### PR TITLE
Add client form and client form metadata sql migration scripts

### PR DESCRIPTION
**Pull request by ekigamba**

### CODE REVIEW

File: assets/migrations/scripts/20200319093703_create_table_client_form_metadata.sql [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+--    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+--    See the License for the specific language governing permissions and
+--    limitations under the License.
+--
+
+-- // create table client_form_metadata
+-- Migration SQL that makes the change goes here.
+CREATE TABLE core.client_form_metadata
+(
+    id INTEGER REFERENCES core.client_form(id),
```
Comment (githengi) [inline assets/migrations/scripts/20200319093703_create_table_client_form_metadata.sql]: should be bigint

Comment (ekigamba) [reply assets/migrations/scripts/20200319093703_create_table_client_form_metadata.sql]: This has been fixed

File: assets/migrations/scripts/20200319093703_create_table_client_form_metadata.sql [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+-- Migration SQL that makes the change goes here.
+CREATE TABLE core.client_form_metadata
+(
+    id BIGINT REFERENCES core.client_form(id),
+    identifier character varying NOT NULL,
+    jurisdiction character varying,
+    version character varying NOT NULL,
+    label character varying(200) NOT NULL,
+    module character varying(200),
+    created_at TIMESTAMP NOT NULL,
```
Comment (githengi) [inline assets/migrations/scripts/20200319093703_create_table_client_form_metadata.sql]: Maybe add a default of current timestamp

File: assets/migrations/scripts/20200319092714_create_table_client_form.sql [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+--    limitations under the License.
+--
+
+-- // create table client_forms
+-- Migration SQL that makes the change goes here.
+CREATE TABLE core.client_form
+(
+    id BIGSERIAL,
+    json jsonb NOT NULL,
+    created_at TIMESTAMP NOT NULL,
```
Comment (githengi) [inline assets/migrations/scripts/20200319092714_create_table_client_form.sql]: Maybe add a default of current timestamp

**Pull request merged**